hack/update-bazel.sh

```diff
@@ -53,7 +53,7 @@ gazelle fix \
     -mode=fix
 
 # Ignore unneeded rebuild for vendored openapi
-kazel | grep -vq 'vendor/k8s.io/kubernetes/pkg/generated/openapi/BUILD' && true
+kazel 3>&1 1>&2 2>&3 3>&- | grep -vq 'vendor/k8s.io/kubernetes/pkg/generated/openapi/BUILD' && true
```

Comment:
some magic here. looks like cyclic redirection. Is it intentional?

Reply:
Yes - this is necessary to allow filtering of stderr.